src/pages/index.js

```diff
@@ -85,43 +127,157 @@ const addModal = new PopUpWithForm(
 editModal.setEventListeners();
 addModal.setEventListeners();
 
+const cardDeletePopUp = new PopUpWithConfirmation({
+  popUpSelector: "#delete-modal",
+});
+
+cardDeletePopUp.setEventListeners();
+
 function handleProfileEditSubmit(inputValues) {
-  userInfo.setUserInfo(inputValues);
-  editModal.close();
+  editModal.setLoading(true);
+  api
+    .setUserInfo(inputValues.title, inputValues.description)
+    .then(() => {
+      userInfo.setUserInfo(inputValues);
+      editModal.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      editModal.setLoading(false);
+    });
 }
 
 function handleImageClick(name, link) {
   previewCardModal.open(name, link);
 }
 
+function handleEditButtonClick() {
+  profileEditModal.classList.add("modal_opened");
+}
+
+if (profileEditButton) {
+  profileEditButton.addEventListener("click", handleEditButtonClick);
+}
```

Comment:
Please, delete it. You have the right code in line 89

Reply:
You haven't deleted something https://github.com/mowooje/se_project_aroundtheus/pull/4#discussion_r1759991746